float64 timestamp
int32 le_us
int32 accuracy
bool valid
position position